docs/fastddsgen/pubsub_app/includes/first_app.rst

```diff
@@ -18,33 +18,33 @@ installation followed and the operating system.
 
     .. code:: bash
 
-        <path-to-Fast-DDS-workspace>/src/fastddsgen/scripts/fastddsgen -example CMake HelloWorld.idl
+        <path-to-Fast-DDS-workspace>/src/fastddsgen/scripts/fastddsgen -no-typeobjectsupport -example CMake HelloWorld.idl
```

Comment:
Do not disable default behavior in a first example. Please apply to other similar commands in this file